metrics/web/lib/project_groups/presentation/models/project_group_persistent_store_error_message.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:metrics/common/domain/entities/persistent_store_error_code.dart';
+import 'package:metrics/project_groups/presentation/strings/project_groups_strings.dart';
+
+/// A class that provides a persistent store error description based
+/// on the [PersistentStoreErrorCode].
+class ProjectGroupPersistentStoreErrorMessage {
```

Comment:
This one seems to be pretty common - shouldn't we try to move it to common? As well as `ProjectGroupsStrings.unknownErrorMessage`?